kolibri/utils/server.py

```diff
@@ -345,7 +348,14 @@ def installation_type():
             if launcher.endswith('.pex'):
                 install_type = 'PEX file'
             elif launcher == '/usr/bin/kolibri':
-                install_type = 'Debian package'
+                install_type = 'Debian package (Manual)'
+                try:
+                    apt_cache = str(check_output(['apt-cache', 'show', 'kolibri']))
+                except CalledProcessError:  # kolibri package not installed!
+                    apt_cache = ''
+                    install_type = 'Executable compiled from code'
+                if '.deb' in apt_cache and 'ilename' in apt_cache:
+                    install_type = 'Debian package (using apt)'
```

Comment:
I have redone it,

Reply:
But `apt-cache` can mis-fire if you have several installations, for instance for developers and testers. This will become confusing.

> Detecting VERSION_PKG would be a better start if we had different versions, but that's not the case as far as I know. 

We do! We may release several builds of the Debian and Windows installers after source releases. Have done this with the Debian installer... it's unfortunately a bit complicated so we do see issues!